smart_card_connector_app/src/background.js

```diff
@@ -151,7 +155,6 @@ executableModule.startLoading();
 if (GSC.Packaging.MODE === GSC.Packaging.Mode.APP)
   chrome.app.runtime.onLaunched.addListener(launchedListener);
 
-chrome.runtime.onConnect.addListener(connectionListener);
```

Comment:
This change seems to affect the APP mode, while the new code only runs in the EXTENSION mode - was this not needed for an app?

Reply:
Correct, it was useless code :see_no_evil: Extracted into #1154.